pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -283,7 +287,26 @@ def generate_state(self, states_only=False):
             p = Pool()
             p.map(self.map_fips, all_fips)
             p.close()
+            p.join()
 
+    def build_own_fips(self, include_fips):
+        state_fips = us.states.lookup(self.state).fips
+
+        df = load_data.load_county_metadata()
+        all_fips = df[df['state'].str.lower() == self.state.lower()].fips
```

Comment:
We should just move this list construction to `pyseir/utils.py` so it doesn't have to be repeated.